### ISSUE TITLE
Improve welcome message on main page

### ISSUE THREAD

**Issue by Shomlings** [NTNUI/opptak#99]
![image](https://user-images.githubusercontent.com/8404794/183307014-65eebf91-1986-4b28-8188-c65d2dcb7cc1.png)
Would it be better to change this text to something like "NTNUI Admin opptak starter DD.MM...."?

**Comment by eivindkopperud** [comment]
Good point, it seems like there won't be a "opptak" at all by the first glance.

What about "Opptaket til NTNU Admin åpner 15. august 2022"?
